diff_drive_controller/src/diff_drive_controller.cpp

```diff
@@ -234,6 +234,24 @@ namespace diff_drive_controller{
       return false;
     }
 
+    // Export parameters to param server if not available
+    if (lookup_wheel_separation)
+    {
+      controller_nh.setParam("wheel_separation", wheel_separation_);
```

Comment:
I'm a little worry about the side effects of setting params from here.

For example, if you re-run the controller, it'd take the params set here, which might be unexpected by some users if they're playing with the lookup values instead.